### ISSUE TITLE
The performance after linear evaluation on ImageNet

### ISSUE THREAD

**Issue by yxgeee** [untitled-ai/self_supervised#2]
Hi,

Thanks for your nice implementation! I am wondering about the performance after training a linear classifier on ImageNet. I noticed a result in your blog (57.7%), but there seems no explanation about the dataset and detailed evaluation method on it.

Looking forward to your reply! Thanks.

**Comment by abefetterman** [comment]
Hi @yxgeee , thanks for the note!

The 57.7% result in the table is linear evaluation accuracy on the STL10 test set. Embeddings from half of the test set are used to fit an `sklearn` linear classifier, and the other half is used for validation. We find that this tracks well with more robust linear evaluation protocols (training a linear layer with SGD). I've added a footnote to make this more clear. Thanks for the comment!
